cmd/utils/flags.go

```diff
@@ -548,6 +548,21 @@ var (
 		Usage: "Maximum allowed number of return items for log collecting filter API",
 		Value: filters.GetLogsMaxItems,
 	}
+	RPCReadTimeout = cli.IntFlag{
+		Name:  "rpcreadtimeout",
+		Usage: "HTTP-RPC server read timeout",
+		Value: 30,
```

Comment:
How about using `DefaultHTTPTimeouts` instead of a static value? 
```suggestion
		Value: rpc.DefaultHTTPTimeouts.ReadTimeout,
```